### PR TITLE
Create RFC for AD CLI

### PR DESCRIPTION
**Pull request by VijayanB**
This is the RFC for the Anomaly Detection CLI feature to be built on AD.

*Issue #, if available:*
N/A

*Description of changes:*
Created RFC to get feedback from community

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by ylwu-amzn** [comment]
Can't add comments on pdf file. How about change to markdown ?

**Comment by wnbts** [comment]
question. how does a user provide comments?

my comments. All anomaly detection functionality is available via rest api. A notebook containing example code for common cases might be another flexible and scalable option.

**Comment by VijayanB** [comment]
@ylwu-amzn Like @wnbts mentioned we will add this(PR) to our odfe roadmap to get feedback.
Nevertheless. i exported to markdown to keep it consistent with read me file.

File: cli/rfc-adcli.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+   Few operations like start, stop, list and delete detector(s) can also be performed using AD CLI
+
+![](Start.png)
+
+**CRUD operation is just a beginning for AD CLI. We can easily identify common use-cases from users which involves multiple operations with AD plugin like copy, export, import , etc... can be easily extracted into commands and be available to the users in multiple releases.**
+
+
+### Request for Comments:
+
+We would like comments and feedback on the proposal for building AD CLI tool. Some specific questions we’re seeking feedback include
```
Comment (ylwu-amzn) [inline cli/rfc-adcli.md:36]: How about we put a Github issue link here, so community can add comments easily.

Comment (VijayanB) [reply cli/rfc-adcli.md]: Created issue and included the link.

**Pull request merged**